graphql/typeDefs.ts

```diff
@@ -152,11 +152,11 @@ export default gql`
   }
 
   type Challenge {
-    id: String
-    description: String
-    lessonId: String
-    title: String
-    order: Int
+    id: String!
```

Comment:
Let's start fixing this stupid thing of ids being strings haha

This will probably break some stuff after generating a new index file, you can then run `yarn type-check` to see what's broken.

```suggestion
    id: Int!
```

Reply:
Good idea, I had to change a bunch of other things (as expected).